PubHub/PubHub.Common/ErrorsCodeConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PubHub.Common
+{
+    public static class ErrorsCodeConstants
+    {
+        public const string NO_CONNECTION = "0x00001";
+        public const string INVALID_TOKEN = "0x00002";
```

Comment:
Just curious. What are thse used for and why in HEX format? 🙂

Reply:
Those are error codes that are displayed for the end user (:
We don't want to publish debug data when an errors occurs, so having a code that is simply for us to read yet complex enough to support large sets of return codes will aid us in tracking down issues.